### ISSUE TITLE
move rio-tiler-mosaic to rio-tiler ?

### ISSUE THREAD

**Issue by vincentsarago** [cogeotiff/rio-tiler-mosaic#16]
because rio-tiler-mosaic is a 0 requirement module maybe we could move it to rio-tiler? 

cc @kylebarron @geospatial-jeff

**Comment by geospatial-jeff** [comment]
Makes sense to me since it mostly maps `rio-tiler` funcs over many images.  Maybe a `rio_tiler.mosaic` module?

**Comment by vincentsarago** [comment]
done https://github.com/cogeotiff/rio-tiler/pull/204

I'm keeping this issue open for documentation
